.github/workflows/release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ steps.get_release.outputs.upload_url }}
-          asset_path: ./sigsci-cloudfoundry-buildpack.tar.gz
-          asset_name: sigsci-cloudfoundry-buildpack.tar.gz
-          asset_content_type: application/gzip
+          asset_path: ./sigsci-cloudfoundry-buildpack.zip
+          asset_name: sigsci-cloudfoundry-buildpack.zip
+          asset_content_type: application/zip
```

Comment:
Is there a reason to not produce .zip and .tar.gz?

Reply:
None. I think I was getting confused from working in pwsh for so long. They only support zip. Changed these back to use gzip.